ionoscloud/data_source_gpu_server.go

```diff
@@ -0,0 +1,186 @@
+package ionoscloud
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func dataSourceGpuServer() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceCubeServerRead,
+		Schema: map[string]*schema.Schema{
```

Comment:
@avirtopeanu-ionos is there a difference between the schema for the `GPU` Server data source and the schema for the `CUBE` Server data source? Maybe we can define the schema somewhere and use it in both places.